dm/tests/tiup/lib.sh

```diff
@@ -269,6 +269,10 @@ function ensure_start_relay() {
 		# always use CUR_VER, because we might use tiup mirror in previous steps.
 		tiup dmctl:$CUR_VER --master-addr=master1:8261 operate-source show -s mysql-replica-01 >$dmctl_log 2>&1
 		worker=$(grep "worker" $dmctl_log | awk -F'"' '{ print $4 }')
-		run_dmctl_with_retry $CUR_VER "start-relay -s mysql-replica-01 $worker" "\"result\": true" 2
+		if [[ "$PRE_VER" == "v2.0.2" ]] || [[ "$PRE_VER" == "v2.0.3" ]] || [[ "$PRE_VER" == "v2.0.4" ]] || [[ "$PRE_VER" == "v2.0.5" ]] || [[ "$PRE_VER" == "v2.0.6" ]]; then
```

Comment:
forget to add v2.0.7 in `previous_v2` :joy: maybe we can only test important versions